app/javascript/src/inscryb-mde.js

```diff
@@ -296,6 +301,44 @@ class InitialiseInscrybeMDE {
     }
   }
 
+  toggleVideoUploader() {
+    const button = this.mde.gui.toolbar.querySelector(".fa-video").closest("button")
+
+    button.classList.toggle("dropdown-open")
+
+    if (button.classList.contains("dropdown-open")) {
+      const dropdownElement = document.createElement("div")
+      dropdownElement.classList.add("editor-dropdown")
+
+      const textElement = document.createElement("small")
+      textElement.innerText = "Upload a video. Limited to mp4 filetype and 50mb filesize."
+
+      const randomId = Math.random().toString().substr(2, 8)
+      const labelElement = document.createElement("label")
+      const labelClasslist = ["button", "button--small", "mt-1/4", "w-100"]
+      labelElement.for = randomId
+      labelElement.innerText = "Upload video"
+      labelElement.classList.add(...labelClasslist)
+
+      const inputElement = document.createElement("input")
+      inputElement.type = "file"
+      inputElement.id = randomId
+      inputElement.accept = "video/mp4"
+      inputElement.classList.add("hidden-field")
+
+      inputElement.addEventListener("change", () => { new InscrybeInsertVideo(event, this.codemirror).input() })
```

Comment:
VSCode is warning me that `event` is deprecated.

```suggestion
      inputElement.addEventListener("change", (event) => { new InscrybeInsertVideo(event, this.codemirror).input() })
```

Technically this works because of funny Browser behaviour (see [`window.event`](https://developer.mozilla.org/en-US/docs/Web/API/Window/event)).